src/components/datalist-item/DatalistItem.scss

```diff
@@ -1,7 +1,9 @@
 @import '../../styles/variables';
 
 .datalist-item {
-    padding: 5px 35px 5px 15px;
+    margin: 0 ($bdl-grid-unit * 2);
```

Comment:
Does this affect other things that could be using data list items?

Reply:
yeah, that is the first screenshot in the "after" section